tests/dummy/app/controllers/application.js

```diff
@@ -1,13 +1,21 @@
 import Controller from 'ember-controller';
 // import fetch from 'fetch';
+import computed from 'ember-computed';
 import RSVP from 'rsvp';
 
 export default Controller.extend({
   // authenticate(/* pw */) {
   //   return fetch('/check-password', {method: 'POST'})
   //     .then(checkStatus);
   // }
-  authenticate: () => RSVP.Promise.resolve()
+  authenticate: () => RSVP.Promise.resolve(),
+  membershipStartDate: computed('model', function() {
+    let memberStatusList = this.get('model.memberStatus'),
+      joinDates = memberStatusList.map(item => item.get('membershipStartDate')),
+      sortedDates = joinDates.sort((a, b) => Date.parse(a) > Date.parse(b)),
+      earliestJoinDate = sortedDates[0];
+    return earliestJoinDate;
+  }),
```

Comment:
if this is a value determined by the server (i.e. it's an attribute on a user model), then I think you ought to compute this in mirage either in the user factory or in the user look up endpoint.